src/app/features/genes/components/gene-evidence-proteomics/gene-evidence-proteomics.component.ts

```diff
@@ -127,31 +120,23 @@ export class GeneEvidenceProteomicsComponent {
         });
 
         if (data) {
-          this.processDifferentialExpressionData(item, data, this.SRMYAxisMin, this.SRMYAxisMax, proteomicData);
+          if (!this.SRMRange)
+            this.SRMRange = new ChartRange(data.min, data.max);
+          this.processDifferentialExpressionData(item, data, this.SRMRange, proteomicData);
         }
       });
 
-      if (this.SRMYAxisMin) {
-        this.SRMYAxisMin -= 0.2;
-      }
-
```

Comment:
Box plot logic moved out of component and into chart.  Having display logic in the component does not make sense.